tests/test_scale.py

```diff
@@ -314,49 +311,12 @@ async def test_scale_cluster_while_create_snapshot_running(
     core = CoreV1Api(api_client)
     name = faker.domain_word()
 
-    await create_fake_cronjob(api_client, name, namespace.metadata.name)
```

Comment:
I would keep these checks, but maybe scale down instead of up, so that it is still interrupted while a snapshot is happening.